src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -306,7 +306,22 @@ class ForwardPort(CommandBase):
 
     def run(self, args, unknown_args):
         environment = get_environment(args.env_name)
+        nice_name = environment.terraform_config.account_alias
         remote_host_key, remote_port = self._SERVICES[args.service]
         loopback_address = f'127.0.{environment.terraform_config.vpc_begin_range}'
         remote_host = lookup_server_address(args.env_name, remote_host_key)
-        return commcare_cloud(args.env_name, 'ssh', 'control', '-NL', f'{loopback_address}:{remote_port}:{remote_host}:{remote_port}')
+        puts(color_notice('Make sure you have done the following:'))
+        puts(color_notice(f'  - Run `sudo ifconfig lo0 alias {loopback_address}`.'))
```

Comment:
Would be nice to include an ansible playbook that devs can run on their workstation to handle this boilerplate in an idempotent way.  It could handle the `/etc/hosts` records and support multiple OSes (e.g. support `ifconfig` and `iproute2`) in an elegant way. Then this comment could be something simpler like:

```python
puts(color_notice('  - Run `ansible-playbook ./.../loopbacks.yml -i localhost --diff --check`.'))
puts(color_notice('    to verify your workstation configuration, or to enforce:'))
puts(color_notice('  - Run `ansible-playbook ./.../loopbacks.yml -i localhost --diff -K`.'))
```

Reply:
Hmm, I see what you're saying. I have a bit of a reaction to the idea of maintaining dev environments with ansible, because everyone does things a little differently and keeping those working right and not interfering with people's individual setups seems like it'll create a really leaky abstraction that's more frustrating than helpful.

Perhaps instead of trying to inline it here, I should link to documentation that has more of a "if you're on Mac do X, if you're on Linux do Y" approach. I don't know. What do you think about my concerns about the leakiness of it?

Reply:
And actually regardless @joxl can you tell me the equivalent command you have to run to get this to work for you locally on Linux? (If you have any trouble getting this set up, ping me on slack because that'll be useful info too.)

Reply:
Yeah, I would not suggest that such a playbook get executed automatically, I would prefer it be up to the developer to do so. But I'm guessing 99% of devs will be doing something very similar (add some loopback aliases and respective host records), so it could be handy.

Linux commands are (Marco also provided this in his email feedback regarding `iproute2`):
```bash
# observe current state of 'lo' interface
$ ip addr show lo
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever

# add an alias for 'lo'
$ sudo ip addr add 127.0.49.201/8 dev lo

# observe change
$ ip addr show lo
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet 127.0.49.201/8 scope host secondary lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever

# remove it (if you wish)
$ sudo ip addr del 127.0.49.201/8 dev lo
```

Reply:
Thanks. I went ahead and made the script detect whether the user has run the setup steps, and then run them through any steps they still have to do interactively. I tried it my self a bunch of times with different setup missing and made sure the experience made sense in every combination.

I'll reach out on slack to have someone test the Linux experience.